src/hooks/useWyreTransferStatus.ts

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback, useEffect } from 'react';
+import { captureException } from '@sentry/react-native';
+import { network as selectedNetwork } from '@models/network';
+import { trackWyreTransfer } from '@models/wyre';
+
+const MAX_TRIES = 10 * 60;
+const MAX_ERROR_TRIES = 3;
+let transferHashHandle: null | ReturnType<typeof setTimeout> = null;
```

Comment:
Is there a reason why you're declaring it outside the function and using the "let" keyword to reassign?